### ISSUE TITLE
Zelda Link's Awakening has color banding on Vulkan

### ISSUE THREAD

**Issue by NineKain** [yuzu-emu/yuzu#7095]
![image](https://user-images.githubusercontent.com/72826232/134827731-ed4f416d-ce92-47dc-b58d-1fd4774ef467.png)

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/7232457/yuzu_log.txt)

The game presents heavy banding under the Vulkan renderer, OpenGL doesn't render as accurately but doesn't have banding.

**Comment by FernandoS27** [comment]
oh yeah, we recall at some point in Hades we kinda fixed it. It's a floating point precision problem. Sadly we need float16 controls to work correctly. WEe even have Ampere blacklisted for Float16 operations.

**Comment by NineKain** [comment]
> oh yeah, we recall at some point in Hades we kinda fixed it. It's a floating point precision problem. Sadly we need float16 controls to work correctly. WEe even have Ampere blacklisted for Float16 operations.

tbh don't really know what that means haha, does that mean it still will take long to fix? should I just switch to OpenGL although it doesn't render as well?

**Comment by vasishath** [comment]
I don't know if its the same issue, but similar color banding is also happening on metroid prime remastered only on vulkan. It can be easily noticed by increasing display brightness. It feels as if the color bit depth is low.

Vulkan:
![Screenshot from 2023-03-17 14-33-38](https://user-images.githubusercontent.com/6087015/225871842-00004186-bf57-4dd2-a38c-5f6f11dd15ad.png)
![Screenshot from 2023-03-17 14-43-08](https://user-images.githubusercontent.com/6087015/225871863-73342aec-9045-4fb4-9352-ee57f9e9c21c.png)


OpenGL:
![Screenshot from 2023-03-17 14-36-35](https://user-images.githubusercontent.com/6087015/225871900-d75303fd-dde0-4212-96b5-9147d1a55001.png)
![Screenshot from 2023-03-17 14-48-11](https://user-images.githubusercontent.com/6087015/225871930-94862f26-bd13-4775-bcdc-3ff4ee870056.png)

**Comment by XxAlcatelxX** [comment]
> No sé si es el mismo problema, pero también están ocurriendo bandas de color similares en Metroid Prime remasterizado solo en Vulkan. Se puede notar fácilmente aumentando el brillo de la pantalla. Se siente como si la profundidad de bits de color fuera baja.
> 
> Mi plataforma: Arch Linux (6.2.6) wayland (también ocurre en X11) NVIDIA MX150 (525.89.02) + Intel UHD 620 (Mesa 22.3.6) Yuzu EA 3459 (también ocurre en la línea principal)
> 
> Vulcano: ![Captura de pantalla del 2023-03-17 14-33-38](https://user-images.githubusercontent.com/6087015/225871842-00004186-bf57-4dd2-a38c-5f6f11dd15ad.png) ![Captura de pantalla de 2023-03-17 14-43-08](https://user-images.githubusercontent.com/6087015/225871863-73342aec-9045-4fb4-9352-ee57f9e9c21c.png)
> 
> OpenGL: ![Captura de pantalla de 2023-03-17 14-36-35](https://user-images.githubusercontent.com/6087015/225871900-d75303fd-dde0-4212-96b5-9147d1a55001.png) ![Captura de pantalla de 2023-03-17 14-48-11](https://user-images.githubusercontent.com/6087015/225871930-94862f26-bd13-4775-bcdc-3ff4ee870056.png)

Hello, that happens because you use FXAA edge smoothing with AMD FSR, which causes that bad effect because it is a very weak and simple edge smoothing, you should change from FXAA to SMAA (more powerful edge smoothing) and your problem will be solved.

**Comment by vasishath** [comment]
> > No sé si es el mismo problema, pero también están ocurriendo bandas de color similares en Metroid Prime remasterizado solo en Vulkan. Se puede notar fácilmente aumentando el brillo de la pantalla. Se siente como si la profundidad de bits de color fuera baja.
> > Mi plataforma: Arch Linux (6.2.6) wayland (también ocurre en X11) NVIDIA MX150 (525.89.02) + Intel UHD 620 (Mesa 22.3.6) Yuzu EA 3459 (también ocurre en la línea principal)
> > Vulcano: ![Captura de pantalla del 2023-03-17 14-33-38](https://user-images.githubusercontent.com/6087015/225871842-00004186-bf57-4dd2-a38c-5f6f11dd15ad.png) ![Captura de pantalla de 2023-03-17 14-43-08](https://user-images.githubusercontent.com/6087015/225871863-73342aec-9045-4fb4-9352-ee57f9e9c21c.png)
> > OpenGL: ![Captura de pantalla de 2023-03-17 14-36-35](https://user-images.githubusercontent.com/6087015/225871900-d75303fd-dde0-4212-96b5-9147d1a55001.png) ![Captura de pantalla de 2023-03-17 14-48-11](https://user-images.githubusercontent.com/6087015/225871930-94862f26-bd13-4775-bcdc-3ff4ee870056.png)
> 
> Hello, that happens because you use FXAA edge smoothing with AMD FSR, which causes that bad effect because it is a very weak and simple edge smoothing, you should change from FXAA to SMAA (more powerful edge smoothing) and your problem will be solved.

Yes, I can confirm this is fixed by either disabling AA or switching to SMAA. But this happens only on vulkan, not on OpenGL.
